### DISCUSSION TITLE
After removing the nvim-tressiter plugin, errors occurred.

### DISCUSSION THREAD

**Discussion by SadSock** [nvim-treesitter/nvim-treesitter#6907] [Q&A]
After I removed nvim-tressiter, an error appeared when opening lua files.

![图片](https://github.com/nvim-treesitter/nvim-treesitter/assets/7643216/940df620-de86-4288-a57a-ac7001fe7ea4)
How should I check the root cause of errors?

**Comment by clason** [comment]
Complain to Gentoo that they need to package the Lua (and other bundled) parsers.

**Reply by SadSock** [reply]
> Complain to Gentoo that they need to package the Lua (and other bundled) parsers.

ok
